pyserini/fusion/__init__.py

```diff
@@ -14,6 +14,6 @@
 # limitations under the License.
 #
 
-from ._base import FusionMethod, interpolation, reciprocal_rank_fusion
+from ._base import average, FusionMethod, interpolation, reciprocal_rank_fusion
 
-__all__ = ['FusionMethod', 'interpolation', 'reciprocal_rank_fusion']
+__all__ = ['average', 'FusionMethod', 'interpolation', 'reciprocal_rank_fusion']
```

Comment:
can you look up to see if there are best practices on how to order these? alphabetical mixed case seems a bit odd to me...

Reply:
I didn't find anything specific on pep8 style guide. But it seems like we should put classes first, e.g. https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/svm/__init__.py